src/main/java/org/kohsuke/github/GHGist.java

```diff
@@ -182,7 +182,7 @@ public void star() throws IOException {
      *             the io exception
      */
     public void unstar() throws IOException {
-        root.createRequest().method("DELETE").withUrlPath(getApiTailUrl("star")).send();
+        root.createRequest().method("DELETE").setRawUrlPath(getApiTailUrl("star")).send();
```

Comment:
This is not a raw URL.  

I'm going to add  check to `setRawUrlPath` the requires it starting with `http`.

Reply:
```suggestion
        root.createRequest().method("DELETE").withUrlPath(getApiTailUrl("star")).send();
```

Reply:
Sounds like a really good idea.